website/source/api/acl/tokens.html.md

```diff
@@ -35,6 +35,12 @@ The table below shows this endpoint's support for
 
 ### Parameters
 
+- `AccessorID` `(string: "")` - Specifies a UUID to use as the token's Accessor ID.
+   If not specified a UUID will be generated for this field. Added in v1.4.3.
+
+- `SecretID` `(string: "")` - Specifies a UUID to use as the token's Secret ID.
+   If not specified a UUID will be generated for this field. Added in v1.4.3.
```

Comment:
This version will need to be updated prior to merge. There is another above as well.